src/systems/odometry_publisher/OdometryPublisher.cc

```diff
@@ -121,6 +134,12 @@ OdometryPublisher::OdometryPublisher()
   {
     std::get<2>(this->dataPtr->linearMean).SetWindowSize(10);
     std::get<2>(this->dataPtr->linearMean).Clear();
+
```

Comment:
Do we need to add back `  std::get<2>(this->dataPtr->angularMean).SetWindowSize(10);` and `  std::get<2>(this->dataPtr->angularMean).Clear();` for the 2D case?

Reply:
You're right, added back the rolling mean for Z-axis angular velocity in [95d88f8](https://github.com/tomcreutz/gz-sim/pull/1/commits/95d88f868b884332673ca52c898d02241ce6e5ec)

I also removed the redundant `Clear` calls as the objects don't have any value at this point.